environments/prod/claim-et-sya-hmcts-net.yml

```diff
@@ -0,0 +1,19 @@
+ns:
```

Comment:
these empty ones need `[]` next to them e.g. `ns: []`

Reply:
Refactored